man/org.freedesktop.systemd1.xml

```diff
@@ -1564,6 +1569,12 @@ node /org/freedesktop/systemd1 {
       file descriptors currently in the file descriptor store of the specified unit. This call is equivalent
       to <function>DumpFileDescriptorStore()</function> on the
       <interfacename>org.freedesktop.systemd1.Service</interfacename>. For further details, see below.</para>
+
+      <para><function>StartAuxiliaryScope()</function> creates a new scope unit from a service where calling
+      process resides. Set of processes that will be migrated to newly created scope is passed as an array
+      of PID file descriptors. This is useful for creating auxiliary scopes that should contain worker
+      processes and their lifecycle shouldn't be bound to a lifecycle of the service, e.g. they should
+      continue running after the restart of the service.</para>
```

Comment:
This should say that `flags` is unused for now

Reply:
also it should say that the main process cannot be migrated, and that processes must be part of the unit requesting the move

Reply:
sorry for the noise, I've pushed what I had resolved so far just to get CI results and I knew it is not complete but I see that automation adds "please-review" after force push.